src/codegen.cpp

```diff
@@ -4988,7 +4990,7 @@ static jl_cgval_t emit_invoke(jl_codectx_t &ctx, const jl_cgval_t &lival, ArrayR
             }
         }
         else {
-            jl_value_t *ci = ctx.params->lookup(mi, ctx.world, ctx.world); // TODO: need to use the right pair world here
+            jl_value_t *ci = ctx.params->lookup(mi, ctx.min_world, ctx.max_world);
```

Comment:
The correct world pair here is the min_world/max_world of the linfo object, which I think is supposed to get propagated to here in `jl_emit_function`?